doc/nrf/protocols/matter/getting_started/adding_clusters.rst

```diff
@@ -360,8 +370,8 @@ To check if the sensor device is working, complete the following steps:
      You can use the :ref:`matter_template_network_mode_onboarding` listed earlier on this page.
    * Send Matter commands.
 
-   At the end of this procedure, the LED indicating the state of the Matter device programmed with the sample starts flashing in the Short Flash Off state.
-   This indicates that the device is fully provisioned, but does not yet have full IPv6 network connectivity.
+   At the end of this procedure, the LED indicating the state of the Matter device programmed with the sample starts presenting Solid On state on Led 1.
```

Comment:
```suggestion
   At the end of this procedure, the LED indicating the state of the Matter device programmed with the sample starts presenting the Solid On state.
```
This would be LED 0 on the nRF54H and nRF54L series devices, so it's easier for us to not name the specific LED.

Reply:
Done